lib/network/modules/InteractionHandler.js

```diff
@@ -165,8 +165,11 @@ class InteractionHandler {
 
   onContext(event) {
```

Comment:
Event `oncontext` triggers on a right-click; your issue seems to suggest that you want to determine the nodes and edges that are hovered over. In that case, events `hoverNode` and `hoverEdge` are more appropriate; these already return the node and edge id's in question.

Please confirm it's really 'oncontext' you need.